### PR TITLE
upgrade to 1.3.1

### PR DESCRIPTION
**Pull request by dam0vm3nt**
upgraded to 1.3.1

### CODE REVIEW

**Comment by jakemac53** [comment]
Would you mind splitting this into two commits, the first one that just updates all the js sources and then a second one with all the actual dart updates? Just makes it a bit easier to process.

**Comment by dam0vm3nt** [comment]
splitted

**Comment by dam0vm3nt** [comment]
actually `1.4` ... I was too optimistic...

File: lib/src/polymer_base.dart [hunk 1]
```diff
@@ -196,12 +196,7 @@ abstract class PolymerBase implements CustomElementProxyMixin {
       jsElement.callMethod('instanceTemplate', [template]);
 
   /// Returns whether a named debouncer is active.
-  bool isDebouncerActive(String jobName) {
-    // TODO(jakemac): Just return once
-    // https://github.com/Polymer/polymer/pull/2291 is submitted.
-    var value = jsElement.callMethod('isDebouncerActive', [jobName]);
-    return value is bool ? value : value != null;
-  }
+  bool isDebouncerActive(String jobName) => jsElement.callMethod('isDebouncerActive', [jobName]);
```
Comment (jakemac53) [inline lib/src/polymer_base.dart]: >80 chars

**Comment by jakemac53** [comment]
Ok, just need a small test for the `activeElement` api now.

File: test/polymer_dom_test.dart [hunk 2]
```diff
@@ -23,6 +23,15 @@ main() async {
       expect(domApi.node, node);
     });
 
+    test('active element', () {
+      PolymerDom rootApi = new PolymerDom(node.root);
+      ButtonElement btn = rootApi.querySelector("#myButton");
+      btn.focus();
+      expect(rootApi.activeElement,btn);
```
Comment (jakemac53) [inline test/polymer_dom_test.dart]: lets add one more expect: `expect(new PolymerDom(document).activeElement, node);`

**Pull request merged**